core/ProfileDetailGenerator.py

```diff
@@ -105,6 +118,20 @@ def filter_element(self, element):
         if element['id'].endswith(".extension"):
             return True
 
+    def check_for_attributes(self, element, attributes_to_check):
```

Comment:
MINOR:
Change function name to reflect behavior "check if at least one of the attributes is present" or
change signature to accept some reduction function (any, all) `fun(list[bool]) -> bool` such that the behavior of the function can be adjusted, e.g.:

```
def check_for_attributes(self, element, attributes_to_check, reduction_fun):
  ...
  if reduction_fun(...):
    return False
  ...
```